test/Bridges/Constraint/geomean.jl

```diff
@@ -9,7 +9,7 @@ const MOIB = MathOptInterface.Bridges
 include("../utilities.jl")
 
 mock = MOIU.MockOptimizer(MOIU.Model{Float64}())
-config = MOIT.TestConfig()
+config = MOIT.TestConfig(duals = false)
```

Comment:
This is not enough, if we add the duals in the test, we need to implement it for `GeoMeanBridge` since otherwise, all solvers that are currently passing GeoMean tests thanks to this bridge will now fail the test

Reply:
the geomean -> RSOC bridge seems more painful to get duals for, given it's slightly awkward structure. do you have any leads on that?

Reply:
@blegat I'm going to leave that to you since you wrote the original bridge. What I can do is add dual solution tests for the other geomean tests in contconic if that's helpful - let me know.